uniref-common/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.uniprot</groupId>
+		<artifactId>uniprot-rest-api</artifactId>
+		<version>1.0.28-SNAPSHOT</version>
+	</parent>
+	<artifactId>uniref-common</artifactId>
+
+	<properties>
+		<maven.compiler.source>11</maven.compiler.source>
```

Comment:
I think we do not need to have compile properties here, it will get from parent pom...  we define it there